core/keyboard_nav/ast_node.js

```diff
@@ -10,38 +10,38 @@
  */
 'use strict';
 
-goog.provide('Blockly.ASTNode');
+goog.module('Blockly.ASTNode');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.connectionTypes');
-goog.require('Blockly.utils.Coordinate');
-
-goog.requireType('Blockly.Block');
-goog.requireType('Blockly.Connection');
-goog.requireType('Blockly.Field');
-goog.requireType('Blockly.IASTNodeLocation');
-goog.requireType('Blockly.IASTNodeLocationWithBlock');
-goog.requireType('Blockly.Input');
-goog.requireType('Blockly.Workspace');
+const Block = goog.requireType('Blockly.Block');
```

Comment:
I think for now you need to add the line disabling the lint warning (sorry, I'm working on it)